beam/core/src/main/java/cz/o2/proxima/beam/direct/io/BlockingQueueLogObserver.java

```diff
@@ -186,7 +194,7 @@ public boolean onNext(StreamElement element, BatchLogObserver.OnNextContext cont
     return enqueue(element, new BatchLogObserverUnifiedContext(context));
   }
 
-  private boolean enqueue(StreamElement element, @Nullable UnifiedContext context) {
+  private boolean enqueue(StreamElement element, UnifiedContext context) {
     try {
       Preconditions.checkArgument(element == null || context != null);
```

Comment:
should this be `element != null && context != null`? (according to new method signature)

Reply:
Yes, I think I fixed this one PR later, but can do it here.